### PR TITLE
Add CellRendererProgress for TreeViews

### PR DESCRIPTION
**Pull request by honorabrutroll**
This is a fairly simple but useful PR, just adds CellRendererProgress to Gir.toml and adds the generated files. The results can be seen [here](https://github.com/honorabrutroll/examples/blob/pending/src/treeview.rs#L19).

### CODE REVIEW

**Comment by honorabrutroll** [comment]
Note: something similar can probably done to add CellRendererPixbuf, which could address issue #213 very easily.

**Comment by gkoz** [comment]
Thanks for taking the generator for a spin! Sadly it doesn't handle properties so these cell renderer objects may still be quite inadequate.
Would you mind trying to add the whole lot of missing `CellRenderer*` objects to`Gir.toml`?
Also the current policy is to keep changes to autogenerated files in separate commits.

**Comment by honorabrutroll** [comment]
In my example it was able to at least render the progress bar with the desired percentage, so I believe the properties aren't that important at the moment except for very specialized use cases. Adding the missing ```CellRenderer*``` objects should be fairly easy. So just to clarify, I would first add the objects to ```Gir.toml``` in one commit and then add the autogenerated files in a separate commit?

**Comment by gkoz** [comment]
Yes, that's right!

**Comment by honorabrutroll** [comment]
Alright, from my searches of the pygtk documentation, I'm getting these ```CellRenderer*``` objects to add in:
* ```CellRendererAccel```
* ```CellRendererCombo```
* ```CellRendererPixBuf```
* ```CellRendererSpin```
* ```CellRendererSpinner```

Does this list look right to you?

**Comment by gkoz** [comment]
The complete hierarchy is here: https://developer.gnome.org/gtk3/stable/ch02.html Looks like there are
```text
    │   ├── GtkCellRenderer
    │   │   ├── GtkCellRendererText
    │   │   │   ├── GtkCellRendererAccel
    │   │   │   ├── GtkCellRendererCombo
    │   │   │   ╰── GtkCellRendererSpin
    │   │   ├── GtkCellRendererPixbuf
    │   │   ├── GtkCellRendererProgress
    │   │   ├── GtkCellRendererSpinner
    │   │   ╰── GtkCellRendererToggle
```

**Comment by honorabrutroll** [comment]
Whups, should've used reset instead of revert.

**Comment by honorabrutroll** [comment]
Fixed.

File: Gir.toml [hunk 1]
```diff
@@ -26,6 +26,12 @@ generate = [
     "Gtk.CellEditable",
     "Gtk.CellLayout",
     "Gtk.CellRendererText",
+    "Gtk.CellRendererAccel",
```
Comment (gkoz) [inline Gir.toml]: Could you preserve the alphabetical order please?

**Comment by gkoz** [comment]
Perfect.
ping @GuillaumeGomez

**Comment by GuillaumeGomez** [comment]
@honorabrutroll: Thanks for you PR!
@gkoz: Thanks for the review (why weren't these files not included at first?)!

**Comment by gkoz** [comment]
I had largely concentrated on not losing the existing APIs rather than including everything to avoid further delaying the merge.

**Pull request merged**